packages.json

```diff
@@ -37,6 +37,7 @@
 				"oddjob-mkhomedir",
 				"opendyslexic-fonts",
 				"playerctl",
+    "podman-machine",
```

Comment:
Per updated discussion with bluefin maintainers... 

1) please move this to the `dx` package list
2) please ensure spacing matches surrounding lines

Then this can be merged.

Thank you!

Reply:
Oh! Alright!